com.civicrm.hrjobroles/js/test/controllers/hr-job-roles-controller/fetching-dates-from-contract_test.js

```diff
@@ -0,0 +1,171 @@
+define([
```

Comment:
These are tests for the `HRJobRolesController` controller, why aren't them in the `hr-job-roles-controller_test.js` file then?